### PR TITLE
Added langchain course mention and youtube multi-modal example

### PR DESCRIPTION
**Pull request by mateig**
- Description: added langchain course mention and youtube multi-modal example to documentation, 
  - Issue: N/A,
  - Dependencies: N/A,
  - Tag maintainer: @baskaryan,
  - Twitter handle: @activeloopai

### CODE REVIEW

**Comment by leo-gan** [comment]
@MateiG Hi , could you, please, resolve the merging issues? After that ping me and I push this PR for the review. Thanks!

**Comment by mgberkeley** [comment]
Hi, I'm no longer affiliated with Activeloop, if you want you can reach out to the maintainers of that repository.

**Comment by leo-gan** [comment]
@baskaryan Can you merge it? This change looks useful.

File: pyproject.toml [hunk 1]
```diff
@@ -181,7 +181,7 @@ pinecone-text = "^0.4.2"
 pymongo = "^4.3.3"
 clickhouse-connect = "^0.5.14"
 transformers = "^4.27.4"
-deeplake = "^3.6.8"
+deeplake = "^3.6.10"
```
Comment (adolkhan) [inline pyproject.toml:184]: same here

File: pyproject.toml [hunk 2]
```diff
@@ -61,7 +61,7 @@ arxiv = {version = "^1.4", optional = true}
 pypdf = {version = "^3.4.0", optional = true}
 networkx = {version="^2.6.3", optional = true}
 aleph-alpha-client = {version="^2.15.0", optional = true}
-deeplake = {version = "^3.6.8", optional = true}
+deeplake = {version = "^3.6.10", optional = true}
```
Comment (adolkhan) [inline pyproject.toml:64]: ?Can we update this to the latest deeplake version, which is 3.7.1?

**Comment by efriis** [comment]
Looks like this one needs a bit more work to land. Happy to reopen if someone wants to take it over. Otherwise I'll close for now :)

**Pull request closed without merging**